gpustack/utils/hub.py

```diff
@@ -40,40 +40,46 @@ def calculate_file_size(
     files: List[FileEntry],
     filename: Optional[str] = None,
     extra_filename: Optional[str] = None,
-) -> int:
+) -> (int, List[str]):
```

Comment:
Returning a file list in a calculate size function can be confusing. Should we separate file matching and size summarizing operations?